municipal_finance/templates/table.html

```diff
@@ -83,6 +83,26 @@
         <ul class="year-chooser list-unstyled"></ul>
       </section>
 
+      <section class="clearfix month">
+        <h5>Month in financial year</h5>
+        <select class="month-chooser form-control">
+          <option value="" selected>Entire year</option>
+          <option value="" disabled="disabled">---</option>
+          <option value="1">July (1)</option>
```

Comment:
@jbothma pls sanity check me here. Month 1 is the first month of the july-june financial year, right?